### ISSUE TITLE
[Bug] When I click the "Load Filter" button in "View" it just disappears.

### ISSUE THREAD

**Issue by sherzinger** [ada-discovery/ada-web#53]
Maybe because I don't have any saved filters? Anyway, this needs some nicer response rather than just disappearing on me :-)

**Comment by peterbanda** [comment]
Yes the functionality is kind of "on demand" :) When you click the button it checks if filters are available.... I agree this hiding game is a bit strange.

There are two options now:

1. Always show the load-filters button and just display "No filters available" if none. 
2. Check initially if filters are available and hide the button if none. Here is a risk as with the current solution that some filters could be added in the meantime.
